libs/liblobby/lobby/lobby.lua

```diff
@@ -600,7 +600,16 @@ function Lobby:_OnUnfriend(userName)
 	self:_CallListeners("OnUnfriend", userName)
 end
 
-function Lobby:_OnFriendList()
+function Lobby:_OnFriendList(friends)
```

Comment:
I think this conflicts with interface_zerok. This is what it does.
```		local newFriendMap = {}
		for i = 1, #friends do
			local userName = friends[i].Name
			if not self.isFriend[userName] then
				self:_OnFriend(userName)
				self:_OnRemoveIgnoreUser(userName)
			end
			newFriendMap[userName] = true
		end

		for _, userName in pairs(self.friends) do
			if not newFriendMap[userName] then
				self:_OnUnfriend(userName)
				self:_OnRemoveIgnoreUser(userName)
			end
		end
```

Reply:
fixed on master